libs/partners/chroma/langchain_chroma/vectorstores.py

```diff
@@ -545,10 +615,10 @@ def max_marginal_relevance_search_by_vector(
         among selected documents.
 
         Args:
-            embedding: Embedding to look up documents similar to.
-            k: Number of Documents to return. Defaults to 4.
-            fetch_k: Number of Documents to fetch to pass to MMR algorithm.
-            lambda_mult: Number between 0 and 1 that determines the degree
+            embedding (List[float]): Embedding to look up documents similar to.
```

Comment:
nit: don't need to add types here, they get read from function signature type hints when api ref docs are built